app/containers/FeaturePage/index.js

```diff
@@ -17,17 +17,15 @@ import styles from './styles.css';
 class FeaturePage extends React.Component {
   constructor() {
     super();
-    this.onChangeRoute = this.onChangeRoute.bind(this);
-    this.changeRouteToHome = this.changeRouteToHome.bind(this);
   }
 
-  onChangeRoute(url) {
+  onChangeRoute = url => {
```

Comment:
Brackets can be skipped when there is only one parameter. It is compulsory when there is either zero or more than one parameter.